bdikit/download.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+import requests
+from tqdm.autonotebook import tqdm
+
+
+default_os_cache_dir = os.getenv(
+    "XDG_CACHE_HOME", os.path.join(os.path.expanduser("~"), ".cache")
+)
+BDIKIT_CACHE_DIR = os.getenv(
+    "BDIKIT_CACHE", os.path.join(default_os_cache_dir, "bdikit")
+)
+BDIKIT_MODELS_CACHE_DIR = os.path.join(BDIKIT_CACHE_DIR, "models")
+
+BUILTIN_MODELS_BOX_URL = {
+    "yurong-reducer-v0.1": "https://nyu.box.com/shared/static/hc4qxzbuxz0uoynfwy4pe2yxo5ch6xgm.pt",
```

Comment:
I would suggest to have another name, e.g. `cl-reducer-v0.1`.

Reply:
done.